src/pages/NotFound/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import DefaultLayout from '../../layout/Default'
+import { Link } from 'react-router-dom'
+
+export default function NotFound() {
+    const { t } = useTranslation('not-found')
+
+    return (
+        <DefaultLayout title={`${t('page-title')} - Instant Visio`}>
```

Comment:
Maybe you could get some styles from https://github.com/Instant-Visio/InstantVisio-WebApp/pull/158/files#diff-0bcbe3f27cb0b8b5e1803367f395ae4c?